### PR TITLE
Standalone phase 2. 

### PR DESCRIPTION
**Pull request by agouin**
Enables heighliner standalone builds:
- Use `heighliner` bin in any working directory, does not need to find `chains.yaml` or any of the dockerfiles from the root of the repo anymore
- Adds `--local` flag which can be used for building from a local directory instead of a remote github repository. Useful for chain development cycles.

Docker requires that the Dockerfile used for building be within the build context directory. It cannot be in a parent directory or other path. To enable local builds, heighliner now has the dockerfiles embedded within the binary. It will look for the local dockerfile, but use the embedded dockerfile if it cannot be found. It will write the applicable dockerfile into a temporary directory within the local repo, then delete after build completion (success of failure).

### CODE REVIEW

File: dockerfile/sdk/local.Dockerfile [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+
+ARG TARGETARCH
+ARG BUILDARCH
+ARG GITHUB_ORGANIZATION
+ARG REPO_HOST
+ARG GITHUB_REPO
+
+WORKDIR /go/src/${REPO_HOST}/${GITHUB_ORGANIZATION}/${GITHUB_REPO}
+
+ADD . .
```
Comment (agouin) [inline dockerfile/sdk/local.Dockerfile:13]: This Dockerfile is the same as native.Dockerfile except for this part

Comment (DavidNix) [reply dockerfile/sdk/local.Dockerfile:13]: This info be really useful as a comment in this file. 😄

File: cmd/build.go [hunk 2]
```diff
@@ -222,41 +307,67 @@ func queueMostRecentReleasesForChain(
...
 		cmdFlags := cmd.Flags()
 
+		configFile, _ := cmdFlags.GetString("file")
+		if configFile == "" {
+			// try to load a local chains.yaml, but do not panic for any error, will fall back to embedded chains.
+			cwd, err := os.Getwd()
+			if err == nil {
+				chainsYamlSearchPath := filepath.Join(cwd, "chains.yaml")
+				if err := loadChainsYaml(chainsYamlSearchPath); err != nil {
+					fmt.Printf("No config found at %s, using embedded chains. pass -f to configure config.yaml path.\n", chainsYamlSearchPath)
```
Comment (DavidNix) [inline cmd/build.go:343]: I'm getting tripped up by the word "embedded" here. It still must be a local file on your filesystem? 

Or have we thought about using `go:embed` to embed the yaml into the binary, so no local files needed?

File: dockerfile/sdk/local.Dockerfile (hunk 1 above)
Comment (agouin) [reply dockerfile/sdk/local.Dockerfile:15]: good call, done

File: cmd/build.go (hunk 2 above)
Comment (DavidNix) [reply cmd/build.go]: Thanks for that clarification. I missed it.

**Review by DavidNix — APPROVED** [review]

**Pull request merged**